src/app/containers/TopStories/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import topStories from '#pages/StoryPage/topStories.json';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+`;
+
+const TopStories = ({ content }) => {
+  const { script, service, dir } = useContext(ServiceContext);
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'top-stories-heading',
+  };
+  const TopStoriesWrapper = ({ children }) => (
+    <Wrapper {...a11yAttributes}>{children}</Wrapper>
+  );
+  TopStoriesWrapper.propTypes = {
+    children: node.isRequired,
+  };
+  if (!topStories.length) return null;
+
+  return (
+    <TopStoriesWrapper>
```

Comment:
Is there a reason why we have 2 wrappers here?

Reply:
I think this is similar to [this comment](https://github.com/bbc/simorgh/pull/5751#discussion_r390855985).

Reply:
Ah yeah, I remember doing the implementation for that. I was going to question why we're using the old grid and not `psammead-grid` but I forgot we're just re-using related-content. Maybe we should make a tech debt issue to use the new grid implementation?

Reply:
This was the PR which added the functionality just for context https://github.com/bbc/simorgh/pull/5584

Reply:
Actually we can get rid of the extra wrapper as we don't need the switching capability as this component is only being used on STY pages atm, so the code can be reduced to this:

```
const TopStoriesWrapper = styled(GridItemConstrainedLarge).attrs({
  as: 'section',
  role: 'region',
  'aria-labelledby': 'top-stories-heading',
})`
  margin-bottom: ${GEL_SPACING_DBL};
  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
    margin-bottom: ${GEL_SPACING_TRPL};
  }
`;

const StyledSectionLabel = styled(SectionLabel)`
  margin-top: 0;
`;

const TopStories = ({ content }) => {
  const { script, service, dir } = useContext(ServiceContext);
  if (!topStories.length) return null;

  return (
    <TopStoriesWrapper>
      <..>
    </TopStoriesWrapper>
```

Reply:
Changes made. Thanks @HarveyPeachey.